neworigins/monsters.cpp

```diff
@@ -41,5 +42,36 @@ void Game::GrowVMons()
 {
 	if (!Globals->LAIR_MONSTERS_EXIST) return;
 
+  std::time_t rawtime;
+  struct tm * timeinfo;
+  char month [60];
+  char day [60];
+
+  time (&rawtime);
+  timeinfo = localtime(&rawtime);
+  strftime(month, 60, "%m", timeinfo);
+  strftime(day, 60, "%d", timeinfo);
+
+  long int day_num = strtol(day, NULL, 10);
+  long int month_num = strtol(month, NULL, 10);
+	
+  // Halloween monsters
+  if (month_num == 10 && day_num > 20) {
+    Awrite("Running Halloween monsters...");
+    forlist(&regions) {
+      ARegion * r = (ARegion *) elem;
+      if (r->type == R_OCEAN) continue;
+
+      int spawn = getrandom(100);
+      if (spawn > 15) continue;
+
+      Faction *mfac = GetFaction(&factions, monfaction);
+      Unit *u = GetNewUnit(mfac, 0);
+      u->MakeWMon("Headless Horseman", I_HHOR, 1);
+      u->MoveUnit(r->GetDummy());
+    }
+    WriteTimesArticle("Headless Horseman has been spotted roaming around...");
```

Comment:
Maybe don't write this as a times article unless a horseman was actually created?